Jenkinsfile

```diff
@@ -68,12 +78,13 @@ node {
                 withCredentials([
                         usernamePassword(credentialsId: 'sonatype', passwordVariable: 'sonatypePassword', usernameVariable: 'sonatypeUsername'),
                         string(credentialsId: 'gpgpassphrase', variable: 'gpgPassPhrase')]) {
-                    sh "${gradleCommand} publish " +
-                            " -Ppublish_username=${sonatypeUsername} " +
-                            " -Ppublish_password=${sonatypePassword} " +
-                            " -PkeyId=DF8285F0 " +
-                            " -Ppassword=${gpgPassPhrase} " +
-                            " -PsecretKeyRingFile=/var/jenkins_home/secring.gpg "
+                    Publish(sonatypeUsername, sonatypePassword, gpgPassPhrase, "apix-interface")
```

Comment:
Calling gradle so many times, every time to publish only one part might not work, because the artifacts may be published to different sonatype repositories.

Instead, you can either

- Run one gradle command with all the projects that need to be published
- Only apply the `maven-publish` plugin to the projects that should be published.

Reply:
Both @koenvangeerteruy and me tried to apply `maven-publish ` selectively first, since it seemed the cleanest, but it didn't work. We ran into problems with the `afterEvaluate` blocks, IIRC.

I can go for the 1st solution though.